examples/semi_supervised/plot_label_propagation_versus_svm_iris.py

```diff
@@ -3,26 +3,27 @@
 Decision boundary of semi-supervised classifiers versus SVM on the Iris dataset
 ===============================================================================
 
-Comparison for the decision boundary generated on the iris dataset
-between Label Propagation, Self-training and SVM.
+A comparison for the decision boundaries generated on the iris dataset
+by Label Propagation, Self-training and SVM.
```

Comment:
Mentions of "label propagation" should be "label spreading" in this example, there's the same discrepancy in the original example